cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -1,119 +1,600 @@
+            frappe.msgprint("Salary Components Added Successfully!")
+
+
+
+
+
+
 
 
 
+    except Exception as e:
+        pass
```

Ruff (0.8.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)